addons/calendar/migrations/11.0.1.0/noupdate_changes.xml

```diff
@@ -0,0 +1,267 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+
+  <record id="ir_cron_scheduler_alarm" model="ir.cron" forcecreate="True">
+    <field name="model"/>
+    <field name="name">Calendar: Event Reminder</field>
+    <field name="function"/>
+    <field name="code">model.get_next_mail()</field>
+    <field name="model_id" ref="model_calendar_alarm_manager"/>
+    <field name="state">code</field>
+  </record>
+
+  <record id="calendar_template_meeting_reminder" model="mail.template">
+    <field name="body_html"><![CDATA[
```

Comment:
When we reset noupdate data during a migration, we need to consider if it would be better if the customer's customizations are retained. This is especially true for report and email templates. If you do reset the data, it would be a good idea to unlink the related translations, otherwise the translated template will still be the old version.

Personally, I think in the case of the calendar reminder templates it would be good to take the new version.